NZSLDict/Classes/SearchViewController.swift

```diff
@@ -398,7 +421,7 @@ class SearchViewController: UIViewController, UISearchBarDelegate, UITableViewDa
     }
 
     func selectEntry(entry: DictEntry) {
-        NSNotificationCenter.defaultCenter().postNotificationName(EntrySelectedName, object: self, userInfo: ["entry": entry])
+        NSNotificationCenter.defaultCenter().postNotificationName(EntrySelectedName, object: nil, userInfo: ["entry": entry])
```

Comment:
@Br3nda my understanding is that the arguments of `postNotificationName` must match what the receiver is expecting for the notification to be triggered. With `object: self`, the received notification had no `userInfo` set, whereas with `object: nil`, the `userInfo` was present.

Like many of these changes, this is just the result of me poking the code until it does what I want.